Jenkinsfile

```diff
@@ -139,31 +45,35 @@ node {
       runTests(testImageName, BUILD_NUMBER)
```

Comment:
Why have `buildTestImage()` and `runTests()` not switched to the shared library?